server/chat-plugins/othermetas.ts

```diff
@@ -542,4 +542,57 @@ export const commands: ChatCommands = {
 	crossevolvehelp: [
 		"/crossevo <base pokemon>, <evolved pokemon> - Shows the type and stats for the Cross Evolved Pokemon.",
 	],
+
+	'!showevo': true,
+	showevo(target) {
+		if (!this.runBroadcast()) return;
+		const targetid = toID(target);
+		if (!targetid) return this.parse('/help showevo');
+		const evo = Dex.getSpecies(target);
+		if (!evo.exists) {
+			return this.errorReply(`Error: Pokemon not found.`);
+		}
+		if (!evo.prevo) {
+			return this.errorReply(`Error: Pokemon does not evolve from another Pokemon.`);
+		}
+		const banlist = Dex.getFormat('gen8crossevolution').restricted || [];
+		if (banlist.includes(evo.name)) {
+			this.errorReply(`Warning: ${evo.name} is banned from being Cross Evolved into.`);
+		}
+		// Fake Pokemon
+		if (evo.isNonstandard === 'CAP') {
+			this.errorReply(`Warning: ${evo.name} is a fake Pokemon created by the CAP Project and is restricted to CAP.`);
+		}
+		const prevoSpecies = Dex.getSpecies(evo.prevo);
+		const deltas = Dex.deepClone(evo);
+		deltas.tier = 'CE';
+		deltas.weightkg = evo.weightkg - prevoSpecies.weightkg;
+		let i: StatName;
+		for (i in evo.baseStats) {
+			const statChange = evo.baseStats[i] - prevoSpecies.baseStats[i];
+			deltas.baseStats[i] = statChange;
+		}
+		deltas.types = [];
+		if (evo.types[0] !== prevoSpecies.types[0]) deltas.types[0] = evo.types[0];
+		if (evo.types[1] !== prevoSpecies.types[1]) {
+			deltas.types[1] = evo.types[1] || evo.types[0];
+		}
+		if (deltas.types.length) {
+			// Undefined type remover
+			deltas.types = deltas.types.filter((type: string | undefined) => type !== undefined);
+
+			if (deltas.types[0] === deltas.types[1]) deltas.types = [deltas.types[0]];
+		} else {
+			deltas.types = null;
+		}
+		const details = {
+			Gen: evo.gen,
+			Weight: (deltas.weighthg < 0 ? "" : "+") + deltas.weighthg / 10 + " kg",
+			Stage: (Dex.getSpecies(prevoSpecies.prevo).exists ? 3 : 2),
+		};
+		const buf = Chat.getDataPokemonHTML(deltas);
+		this.sendReply(`|raw|<div class="message"><ul class="utilichart">${buf}<li style="clear:both"></li></ul></div>`);
```

Comment:
```suggestion
		this.sendReply(`|raw|${Chat.getDataPokemonHTML(deltas)}`);
```
The html for this is already included in the function

Reply:
i completely forgot oops